src/NzbDrone.Core/Parser/Parser.cs

```diff
@@ -22,6 +22,10 @@ public static class Parser
 //                new Regex(@"^(?:(?<absoluteepisode>\d{2,3})(?:_|-|\s|\.)+)+(?<title>.+?)(?:\W|_)+(?:S?(?<season>(?<!\d+)\d{1,2}(?!\d+))(?:(?:\-|[ex]|\W[ex]){1,2}(?<episode>\d{2}(?!\d+)))+)",
 //                          RegexOptions.IgnoreCase | RegexOptions.Compiled),
 
+                //Serie with Title - Some text and season as number, Single ([Cap.105]) AND Multi ([Caps.105_107])
+                new Regex(@"^(?<title>.+?)(\s*\-\s*)(\w+)(\s*|.)?(\s*|.)?(?<!\d)(?<season>\d+)?(.+?)(\[Cap(\w*?)(.))(?<!\d)(\k<season>(?<episode>\d+))((_\k<season>(?<episode>\d+)\])|(.*))",
```

Comment:
This seems very specific, but I don't really see why its needed given the short example here.

A few things:

- Tests need to be added for this regex, for both single and multi-episode conventions.
- Season/episode numbers need to exclude other leading/trailing episode numbers
- I don't think this should be above all the anime regexes either, but tests will confirm